persistence/scaladsl/src/main/scala/com/lightbend/lagom/internal/scaladsl/persistence/PersistentEntityActor.scala

```diff
@@ -201,6 +201,12 @@ private[lagom] class PersistentEntityActor(
 
     case PersistentEntityActor.Stop =>
       context.stop(self)
+
+    /* The actor will receive a SaveSnapshotSuccess message when a snapshot
+       * was successfully saved. There's no need to do anything, but if we don't
+       * handle it, it will pollute the event stream with UnhandledMessage
+       * notifications */
+    case SaveSnapshotSuccess(_) => () // nothing to do
   }
```

Comment:
This is great @lutzh !

I think we'll need the same change in the `PersistentEntityActor` for the `javadsl` (meh, some classes are duplicate for scaladsl and javadsl:-( )